### ISSUE TITLE
[Q] Registration and Login

### ISSUE THREAD

**Issue by ghost** [DevStudents/EPeople-Web#4]
Hey
I would like to start here on something to work, can the main founder of the repository describe how he wants to have written registration and login in php? It needs information about what data is required for registration and what to use to log in.

**Comment by SKocur** [comment]
Hi @f1ndev 

### (my vision) To register, user will have to submit his:
* username (must be unique)
* name
* surname
* date birth
* e-mail (must be unique)
* password

To login only `e-mail` and `password` will be required. If you have another idea please tell about it.

**Comment by cyntler** [comment]
Good vision by @SKocur in my opinion.
Can we write an application on the Symfony 4.0 framework?
I started to prepare the project.

**Comment by SKocur** [comment]
@sintloer
Yes, we can use Symfony if you are familiar with it. I am Java/Android Dev so I don't have such great knowledge about PHP frameworks as you, I think. ;)

**Comment by ghost** [comment]
Which technology will You propose for front part?

**Comment by SKocur** [comment]
Maybe something which allows creating responsive UI (mobile, pc) like Bootstrap?
